Packs/WALLIXBastion/Integrations/WAB/WAB.yml

```diff
@@ -3587,20 +4706,60 @@ script:
     - name: notification_post_enabled
       description: Notification is enabled.
       required: true
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
     - name: notification_post_type
       description: Notification type.
       required: true
+      auto: PREDEFINED
+      predefined:
+      - email
```

Comment:
if email is the only option- why do we need this argument?

Reply:
The API was designed with the intent that the list of notification types would grow in the future